src/main/resources/application.yaml

```diff
@@ -15,6 +15,10 @@ elastic:
   password: ${ZEEBE_ELASTIC_PASSWORD}
   index: ${ZEEBE_ELASTIC_INDEX}
 
+expiration:
+  processInstances:
+    completed: ${EXPIRATION_PROCESS_INSTANCES_COMPLETED_DAYS}
```

Comment:
could you add a default value here for the case when env variable is not set? For example 14 days